x/auth/src/params.rs

```diff
@@ -197,3 +197,83 @@ impl<PSK: ParamsSubspaceKey> ParamsKeeper<PSK> for AuthParamsKeeper<PSK> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use gears::{
+        context::init::InitContext,
+        params::SubspaceParseError,
+        store::{bank::multi::ApplicationMultiBank, database::MemDB, StoreKey},
+    };
+
+    use super::*;
+
+    #[test]
+    fn app_hash() {
+        let keeper = AuthParamsKeeper {
+            params_subspace_key: SubspaceKey::Auth,
+        };
+
+        let mut multi_store = ApplicationMultiBank::<_, SubspaceKey>::new(MemDB::new());
+
+        let before_hash = multi_store.head_commit_hash();
+
+        let mut ctx = InitContext::new(
```

Comment:
Here you've used `gears` with `utils` feature. Please, reuse or update and reuse helper.